webapp/pages/post/_id/_slug/index.vue

```diff
@@ -122,29 +122,6 @@ export default {
       this.title = this.post.title
     },
   },
-  async asyncData(context) {
-    const {
-      params,
-      error,
-      app: { apolloProvider, $i18n },
-    } = context
-    const client = apolloProvider.defaultClient
-    const query = PostQuery($i18n)
-    const variables = { slug: params.slug }
-    const {
-      data: { Post },
-    } = await client.query({ query, variables })
-    if (Post.length <= 0) {
-      // TODO: custom 404 error page with translations
-      const message = 'This post could not be found'
-      return error({ statusCode: 404, message })
```

Comment:
Removing `asyncData` will also remove the feature of sending a 404 if there is no post!

Reply:
I don't think that is true
![Screenshot at 2019-08-16 13:04:24](https://user-images.githubusercontent.com/26943915/63163687-707c5f00-c026-11e9-8ad5-1fb82919e663.png)